arlo-client/src/components/Audit/Setup/Review/index.tsx

```diff
@@ -40,22 +40,26 @@ const Review: React.FC<IProps> = ({ prevStage, locked, refresh }: IProps) => {
   const [{ electionName, randomSeed, riskLimit, online }] = useAuditSettings(
     electionId!
   )
-  const participants = useParticipantsApi(electionId!)
+  const jurisdictions = useJurisdictions(electionId!)
   const [{ contests }] = useContestsApi(electionId!, true)
   const targetedContests = contests
     .filter(c => c.isTargeted === true)
     .map(c => ({
       ...c,
       jurisdictionIds: c.jurisdictionIds.map(j =>
-        participants.length > 0 ? participants.find(p => p.id === j)!.name : ''
+        jurisdictions.length > 0
+          ? jurisdictions.find(p => p.id === j)!.name
+          : ''
```

Comment:
I think this is n^2 in the number of jurisdictions? Should we instead have an object of jurisdictions keyed by `id`? Also, the conditional on `jurisdictions.length` doesn't need to be inside the `map`.

Reply:
I just renamed a variable here. do you want me to make this change?

Reply:
oh hah! I thought you'd introduced this. No, then maybe file a followup ticket to investigate this?

Reply:
sure: https://github.com/votingworks/arlo/issues/474